.golangci.yml

```diff
@@ -16,14 +16,10 @@ linters:
   disable:
     - lll
 run:
-  skip-files:
-    - schema.go
-  skip-dirs:
-    - provider/pkg/gen
-  skip-dirs-use-default: true
   modules-download-mode: readonly
   timeout: 10m
 issues:
+  exclude-dirs-use-default: true
   exclude-rules:
-    path: "provider/cmd/pulumi-resource-kubernetes"
-    source: "^//go:embed"
+    - path: "provider/cmd/pulumi-resource-kubernetes"
+      source: "^//go:embed"
```

Comment:
We could probably also remove these lines. We previously needed this to prevent linting errors when the `schema-embed.json` file was missing. This occurs when we checkout the repo and haven't generated these files yet. I believe golangci-lint allowed ignoring these errors in the past, but they are now build errors, so it is impossible to ignore them as linting requires the Go project to be built first.

We now have this step to disable go embeds during linting. https://github.com/pulumi/pulumi-kubernetes/blob/14f15e9eb754acb5ed7a9637103ed3ac053e9960/.github/workflows/run-acceptance-tests.yml#L537-L539

Reply:
More information about that here:
https://github.com/golangci/golangci-lint/discussions/4984

For posterity:
```
❯ rm cmd/pulumi-resource-kubernetes/schema-embed.json
❯ golangci-lint cache clean

❯ golangci-lint run -c ../.golangci.yml cmd/pulumi-resource-kubernetes
cmd/pulumi-resource-kubernetes/main.go:28:12: pattern schema-embed.json: no matching files found (typecheck)
//go:embed schema-embed.json
```